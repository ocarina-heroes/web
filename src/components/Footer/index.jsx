import styled from 'styled-components'

const Footer = styled.footer`
  width: 100%;
  height: 100px;
  border-top: 1px solid #eaeaea;
  display: flex;
  justify-content: center;
  align-items: center;
  img {
    margin-left: 0.5rem;
  }
  a {
    display: flex;
    justify-content: center;
    align-items: center;
  }
`

export default () => (
  <Footer>
    <a
      href="https://zeit.co?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
      target="_blank"
      rel="noopener noreferrer"
    >
      Powered by <img src="/zeit.svg" alt="ZEIT Logo" />
    </a>
  </Footer>
)
