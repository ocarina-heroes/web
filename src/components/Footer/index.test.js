import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import Footer from '.'

test('it matches snapshot', () => {
  const tree = renderer.create(<Footer />).toJSON()
  expect(tree).toMatchSnapshot()
})
