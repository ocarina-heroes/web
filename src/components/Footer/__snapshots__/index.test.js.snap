// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it matches snapshot 1`] = `
.c0 {
  width: 100%;
  height: 100px;
  border-top: 1px solid #eaeaea;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c0 img {
  margin-left: 0.5rem;
}

.c0 a {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<footer
  className="c0"
>
  <a
    href="https://zeit.co?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
    rel="noopener noreferrer"
    target="_blank"
  >
    Powered by 
    <img
      alt="ZEIT Logo"
      src="/zeit.svg"
    />
  </a>
</footer>
`;
